docs/en/integrations/language-clients/java/index.md

```diff
@@ -6,761 +6,118 @@ description: Options for connecting to ClickHouse from Java
+|Int8                   |✔                    |✔                    |
+|Int16                  |✔                    |✔                    |
+|Int32                  |✔                    |✔                    |
+|Int64                  |✔                    |✔                    |
+|Int128                 |✔                    |✔                    |
+|Int256                 |✔                    |✔                    |
+|UInt8                  |✔                    |✔                    |
+|UInt16                 |✔                    |✔                    |
+|UInt32                 |✔                    |✔                    |
+|UInt64                 |✔                    |✔                    |
+|UInt128                |✔                    |✔                    |
+|UInt256                |✔                    |✔                    |
+|Float32                |✔                    |✔                    |
+|Float64                |✔                    |✔                    |
+|Decimal                |✔                    |✔                    |
+|Decimal32              |✔                    |✔                    |
+|Decimal64              |✔                    |✔                    |
+|Decimal128             |✔                    |✔                    |
+|Decimal256             |✔                    |✔                    |
+|Bool                   |✔                    |✔                    |
+|String                 |✔                    |✔                    |
+|FixedString            |✔                    |✔                    |
+|Nullable               |✔                    |✔                    |
+|Date                   |✔                    |✔                    |
+|Date32                 |✔                    |✔                    |
+|DateTime               |✔                    |✔                    |
+|DateTime32             |✔                    |✔                    |
+|DateTime64             |✔                    |✔                    |
+|Interval               |✗                    |✗                    |
+|Enum                   |✔                    |✔                    |
+|Enum8                  |✔                    |✔                    |
+|Enum16                 |✔                    |✔                    |
+|Array                  |✔                    |✔                    |
+|Map                    |✔                    |✔                    |
+|Nested                 |✔                    |✔                    |
+|Tuple                  |✔                    |✔                    |
+|UUID                   |✔                    |✔                    |
+|IPv4                   |✔                    |✔                    |
+|IPv6                   |✔                    |✔                    |
+|Object                 |✗                    |✔                    |
+|Point                  |✔                    |✔                    |
+|JSON                   |✔                    |✔                    |
```

Comment:
`JSON` is reserved for the new JSON type. I don't think either client supports it.

Reply:
Removed